### ISSUE TITLE
Can YouTubeCenter hide/remove the channel branding "bug" overlays that some videos have?

### ISSUE THREAD

**Issue by Fofer** [YePpHa/YouTubeCenter#1854]
These channel branding "bug" overlays on so many YouTube videos can be just as distracting as they've been on televisions for years now:  http://www.illumina.net/default.htm

See screenshot, from SNL's Black Widow Trailer (parody) here:

![black widow trailer - www youtube com watch v j_5kgpn38hm](https://cloud.githubusercontent.com/assets/518749/7869228/384d3618-0536-11e5-86a4-dcca76626092.png)

http://imgur.com/gallery/AKt6Afs/

Original YouTube video here:  http://www.youtube.com/watch?v=j_5KgpN38hM

Can YouTubeCenter have a setting to remove/hide these "bugs," please?  SPOI has this, but now that SPOI is conflicting with YTC, I had to make the choice (and of course YTC wins.)  This is the only feature I'm seriously missing now.  

I'd appreciate the consideration.  Many thanks!

**Comment by SuperSajuuk** [comment]
Why are you calling them bugs? These aren't bugs, just disable annotations and they'll be removed from the video.

**Comment by Fofer** [comment]
Gotcha - I appreciate it! They reminded me of TV logo "bugs" as described here: 
http://en.wikipedia.org/wiki/Digital_on-screen_graphic
http://www.illumina.net/default.htm

I clearly had no idea that YTC was referring to them as "annotations."  Thanks so much!

**Comment by SINE** [comment]
Please close the issue!

**Comment by Fofer** [comment]
These annotations are still appearing for some videos, even with "enable annotations" unchecked.

Example video:  https://www.youtube.com/watch?v=R1nfAxkpx9M

**Comment by SuperSajuuk** [comment]
@Fofer No idea if you are still watching this, but you can control those particular annotations with a setting located here: https://www.youtube.com/account_playback

**Comment by Fofer** [comment]
@SuperSajuuk Yes, I am still watching, and thanks for that link!  I wasn't aware of it, obviously, and very happy to have "native" controls in YouTube's settings to turn (at least some of?) them off.   

So is YTC's "enable annotations" checkbox now redundant?  Or is it meant to cover more/other bases?

I do still contend that YTC's setting should be the reverse.  I think it would be more intuitive to have us check the box to DISABLE them (as opposed to checking the box to enable them, per YouTube's default state.)

Thanks again!

**Comment by SuperSajuuk** [comment]
@Fofer Those options have been there for years, but Google's updates over the last 1-2 years have resulted in a lot of the older "account" style pages virtually impossible to reach now.

I guess that YT Center currently only supports removal of normal, user-added annotations. The parts covering channel promotions and interactive cards don't seem to be getting tracked by YT Center and is likely due to the fact that these aren't really annotations in the sense.

The channel promotions only run for a short time anyway, so it's somewhat pointless for them to be removed since I believe they run for about 10-15 seconds maximum. The interactive cards is a relatively new feature and I guess @YePpHa hasn't had a chance to fix that bug.

It would be nice if there were MORE developers involved with YT Center though: just @YePpHa being involved in making commits means that updates to the script are so lengthy that bugs are going unfixed for ages. There is no chance of the bugs listed here being cleaned up. It would be nice if @Yonezpt had the right to make commits to the repository, but until that happens... don't expect many updates.

**Comment by LinAGKar** [comment]
Disabling annotations isn't really a good way to do this since you'll, well, lose annotation. I would like an option to remove only the "bug". Also looking at the "Remove Branding Watermark" it sounds like it would to this, but it doesn't seem to, so what does it do?

**Comment by ireun** [comment]
@Fofer Have you tried [Removing watermarks](https://github.com/YePpHa/YouTubeCenter/wiki/Features#Remove_Branding_Watermark) ?

**Comment by LinAGKar** [comment]
Yes, I have that active.

Den 2015-07-16 20:59, IreuN skrev:

**Comment by Fofer** [comment]
Me too.  Doesn't seem to make a universal difference.  I still get them, from time to time, on various videos.  I'm not sure I fully understand the difference between "channel branding," "annotations," and "watermarks" but ideally I'd see none of them.  

YTC used to block them all, but with YouTube constantly changing things, this feels like a game of "Whack-a-Mole."
